multimodal/tests/unittests/others_2/test_hpo.py

```diff
@@ -8,9 +8,9 @@
 from autogluon.multimodal import MultiModalPredictor
 
 from ..predictor.test_predictor import verify_predictor_save_load
-from .test_matcher import verify_matcher_save_load
-from .unittest_datasets import IDChangeDetectionDataset, PetFinderDataset
-from .utils import get_home_dir
+from ..others.test_matcher import verify_matcher_save_load
```

Comment:
are we sure this is a good practice? Why not have utils in a separate folder altogether from `other` `other_2` etc.?

I'd imagine ideally tests shouldn't be importing other tests.

Reply:
Yea, this probably should live in some util files or being a function in conftest @zhiqiangdon .
Though I think it's better to address it in a separate PR.

With that being said, I should move utils and unittest_datasets into a util folder indeed

Reply:
Good point. I can refactor this later.